src/test/resources/allure.properties

```diff
@@ -0,0 +1,9 @@
+#
+# Copyright Skodjob authors.
+# License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+#
+
+allure.results.directory=target/allure-results
+
+allure.link.issue.pattern = https://issues.redhat.com/browse/{}
```

Comment:
Why do you put redhat url into upstream project?

Reply:
this tracker is used in upstream project, https://github.com/search?q=org%3Aopendatahub-io%20%22issues.redhat.com%22&type=code